pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -377,6 +377,8 @@ func (s *Reconciler) Exists(ctx context.Context) (bool, error) {
 		klog.Infof("Machine %v is running", to.String(vm.VMID))
 	case v1beta1.VMStateUpdating:
 		klog.Infof("Machine %v is updating", to.String(vm.VMID))
+	case v1beta1.VMStateCreating:
```

Comment:
Can we add a link here to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/states-lifecycle#provisioning-states

Is there any other provisioning state we want to include here e.g deleting?